src/mono/sample/wasm/browser-threads-minimal/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<!--  Licensed to the .NET Foundation under one or more agreements. -->
+<!-- The .NET Foundation licenses this file to you under the MIT license. -->
+<html>
+
+<head>
+    <title>Wasm Browser Smoke Test Threading Sample</title>
```

Comment:
Just curious, how is the browser-threads-minimal sample different from browser-threads ?

Reply:
So in my mind `browser-threads` is meant to be an interactive sample. So there's UI fluff and less focus on giving a correct answer. 

`browser-threads-minimal` is meant to be a smoketest - so right now it just starts background work in 2 different ways and verifies that it actually ran and computed something correctly.  In the future it could also have some smoke tests for  calling background JS or doing something that verifies that Monitor, Semaphore, etc are working properly.

Reply:
My2c:
In the future I would prefer regular unit test project in libraries, even as a smoke test. Samples are for experiments.